config/nginx-default.conf

```diff
@@ -0,0 +1,28 @@
+server {
+    server_name localhost;
+    listen 80;
+    root /srv/annotations/web;
+
+    location / {
+        try_files $uri /app_dev.php$is_args$args;
+    }
+
+    location ~ ^/app_dev\.php(/|$) {
+        fastcgi_split_path_info ^(.+\.php)(/.+)$;
+        include fastcgi_params;
+        fastcgi_param SCRIPT_FILENAME $request_filename;
+        fastcgi_param ENVIRONMENT_NAME dev;
+        fastcgi_intercept_errors on;
+        fastcgi_pass php:9000;
+        internal;
+    }
+
+    # return 404 for all other php files not matching the front controller
+    # this prevents access to other php files you don't want to be accessible.
+    location ~ \.php$ {
+        return 404;
+    }
+
+    access_log /var/log/nginx/access.log combined;
```

Comment:
should these be going to stdout/err ?

Reply:
Yes, that seems the easiest. They would be merged into a single stream which is weird, but it seems even in Kubernetes the granularity of logs would be per-container.

An update on application logs is that (if permissions and sharing of the filesystem can be figured out) a sidecar container can be used to tail the application log files: https://kubernetes.io/docs/concepts/cluster-administration/logging/#streaming-sidecar-container